app/views/users/confirmations/confirmed.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="govuk-grid-row">
+  <h2 class="govuk-heading-m confirmed-banner">Your school is registered</h2>
```

Comment:
This wants to be `<%= govuk_panel title: "Your school is registered", body: nil %>`

Reply:
@ethanmills does this apply to all the page titles?

Reply:
No, this gives you a [panel](https://design-system.service.gov.uk/components/panel/)